apps/glusterfs/app_config.go

```diff
@@ -33,8 +33,9 @@ type GlusterFSConfig struct {
 	AverageFileSize uint64 `json:"average_file_size_kb"`
 
 	//block settings
-	CreateBlockHostingVolumes bool `json:"auto_create_block_hosting_volume"`
-	BlockHostingVolumeSize    int  `json:"block_hosting_volume_size"`
+	CreateBlockHostingVolumes bool   `json:"auto_create_block_hosting_volume"`
+	BlockHostingVolumeSize    int    `json:"block_hosting_volume_size"`
+	BlockHostingVolumeOptions string `json:"block_hosting_volume_options"`
```

Comment:
I'm not in love with cramming more stuff into the global config, but it's ok as a short term thing.